checkov/terraform/checks/resource/gcp/PubSubPrivateTopic.py

```diff
@@ -0,0 +1,40 @@
+
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class PubSubPrivateTopic(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that Pub/Sub Topics are not anonymously or publicly accessible"
+        id = "CKV_GCP_99"
+        supported_resources = ['google_pubsub_topic_iam_member', 'google_pubsub_topic_iam_binding']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
```

Comment:
feels like I have seen this code somewhere :D

Reply:
Haha yeah, a lot of these `iam_member` and `iam_binding` checks can use the same leverage. Once we got one working it was much easier to replicate for other resource types.